src/registrar/config/settings.py

```diff
@@ -529,8 +529,9 @@
 SESSION_COOKIE_NAME = "JSESSIONID"
 
 # Prevents session cookie from being sent if the user
```

Comment:
**suggestion (non-blocking):** Can we change this comment to reflect the new value of the setting? "Do not prevent session cookie ..."?